app/api/server/v1/voip/omnichannel.ts

```diff
@@ -0,0 +1,94 @@
+import { Match, check } from 'meteor/check';
+
+import { API } from '../../api';
+import { Users } from '../../../../models/server/index';
+import { hasPermission } from '../../../../authorization/server/index';
+import { LivechatVoip } from '../../../../../server/sdk';
+
+API.v1.addRoute('omnichannel.agent.extension', { authRequired: true }, {
+	// Get the extensions associated with the agent passed as request params.
+	get() {
+		if (!hasPermission(this.userId, 'query-agent-extension-association')) {
+			return API.v1.unauthorized();
+		}
+		check(this.requestParams(), Match.ObjectIncluding({
+			username: String,
+		}));
+		const user = Users.findOneByAgentUsername(this.requestParams().username, {
+			fields: { _id: 1 },
+		});
+		if (!user) {
+			return API.v1.notFound();
```

Comment:
```suggestion
			return API.v1.notFound('User not found');
```
Trying to be explicit here since this method returns 2 no found errors

Reply:
Fixed.